ibis_substrait/compiler/translate.py

```diff
@@ -22,11 +22,14 @@
 import ibis.expr.schema as sch
 import ibis.expr.types as ir
 import toolz
+from google.protobuf.any_pb2 import Any as pb2_any
 from ibis import util
 from ibis.util import to_op_dag
 
+from ..proto import extension_rels_pb2 as extrel
```

Comment:
This seems like a leftover since it's being redefined on line 32, that said, any proto thing that isn't namespaced into `substrait.ibis` can potentially collide with any C++ consumer because `protobuf`.  (It looks like this will be a non-issue, though)